src/java.base/share/classes/jdk/internal/module/ModuleBootstrap.java

```diff
@@ -268,7 +268,8 @@ private static ModuleLayer boot2() {
         if (baseUri == null)
             throw new InternalError(JAVA_BASE + " does not have a location");
         BootLoader.loadModule(base);
-        Modules.defineModule(null, base.descriptor(), baseUri);
+        SharedSecrets.getJavaLangAccess()
+                .addEnableNativeAccess(Modules.defineModule(null, base.descriptor(), baseUri));
```

Comment:
This would be cleaner if you replace it with:

Module baseModule = Modules.defineModule(null, base.descriptor(), baseUri);
JLA.addEnableNativeAccess(baseModule);

You can use JLA in addEnableNativeAccess too, no need for "jla".

Reply:
ok